python/tk_multi_breakdown2/file_model.py

```diff
@@ -291,7 +296,7 @@ def data(self, role):
             """
 
             if role == QtCore.Qt.BackgroundRole:
-                return QtGui.QApplication.palette().midlight()
+                return UI_CONFIG_ADV_HOOK.get_item_background_color()
```

Comment:
Since this is only returning the hook function's return value, we can simplify this by removing this `if` case and add the following line to the `role_methods` dictionary (defined on line `517`):
```
self.role_methods = {
    ...
    QtCore.Qt.BackgroundRole: UI_CONFIG_ADV_HOOK.get_item_background_color
}
```

Reply:
Yeah that's better indeed! I've updated!